coderd/autobuild/lifecycle_executor.go

```diff
@@ -177,6 +178,15 @@ func (e *Executor) runOnce(t time.Time) Stats {
 				err := e.db.InTx(func(tx database.Store) error {
 					var err error
 
+					ok, err := tx.TryAcquireLock(e.ctx, database.GenLockID(fmt.Sprintf("lifecycle-executor:%s", wsID)))
+					if err != nil {
+						return xerrors.Errorf("try acquire lifecycle executor lock: %w", err)
+					}
+					if !ok {
+						log.Debug(e.ctx, "unable to acquire lock for workspace, skipping")
```

Comment:
Should we add wsID as a field? Seeing a bunch of these without knowing which workspace is affected could be confusing.

Reply:
Already handled by the definition of `log`

https://github.com/coder/coder/blob/c45817bc62a3e408310fab7818876ee7e546a4a4/coderd/autobuild/lifecycle_executor.go#L161-L164